packages/deploy/deploy/015_land/010_deploy_polygon_land.ts

```diff
@@ -0,0 +1,28 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+  const {deployer, upgradeAdmin} = await getNamedAccounts();
+
+  const TRUSTED_FORWARDER = await deployments.get('TRUSTED_FORWARDER_V2');
+  await deploy('PolygonLand', {
+    from: deployer,
+    contract:
+      '@sandbox-smart-contracts/core/src/solc_0.8/polygon/child/land/PolygonLandV2.sol:PolygonLandV2',
```

Comment:
why doesn't it deploy from the new land package ?
the core package files will eventually be deleted

Reply:
because we want to compare the current deploy (which is from core) with the one from the new `land` package